### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_Proxiestab _Keyboard: Table is not acccesable through keyboard

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5617]
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS2.1.1; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
Tool Name: Keyboard
MAS Reference: MAS2.1.1

**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the connected server->Select the database->Navigate to the SQL agent tab.
4.Navigate to the proxies tab->Hit the tab button to navigate in table.
5. Observed that table is not accessible through keyboard.

**Actual:**
Table is not accessible through keyboard.

**Similar issue is observed for the below pages:**
**1.Jobs:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->
Navigate to jobs page.
**2.Operatortab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Operator tab
**3. Alertstab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Alerts tab.
**4.CreateAlertResponse:** Go to the SQL server agent tab->On alerts tab->Click on new alert button->Select response tab.
**5.NewJobSchedule:** Navigate to the SQL agent tab from Home page->Click on new job button->Select schedule tab.
**6.NewJobAlerts:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Click on New Job button-> Navigate to Alerts tab.     

**Expected:**
Table should  accessible through keyboard.

**User Impact:**
User with disabilities will not be able to navigate through table. 

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
[5617_[Accessibility]A11y__Azure Data Studio_Proxiestab _Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3216517/5617_.Accessibility.A11y__Azure.Data.Studio_Proxiestab._Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes
